libraries/chain/controller.cpp

```diff
@@ -624,11 +630,22 @@ struct controller_impl {
 
       trace->elapsed = fc::time_point::now() - trx_context.start;
```

Comment:
This is already done above. Also the comment is wrong as only hard failure logic is below.